tools/slang-unit-test/unit-test-coverage-cpu-runtime.cpp

```
@@ -0,0 +1,348 @@
+        linked->getEntryPointMetadata(0, 0, metadata.writeRef(), diagnostics.writeRef()) ==
+        SLANG_OK);
+
+    auto coverage = (slang::ICoverageTracingMetadata*)metadata->castAs(
+        slang::ICoverageTracingMetadata::getTypeGuid());
+    SLANG_CHECK_ABORT(coverage != nullptr);
+    auto syntheticResources = (slang::ISyntheticResourceMetadata*)metadata->castAs(
+        slang::ISyntheticResourceMetadata::getTypeGuid());
+    SLANG_CHECK_ABORT(syntheticResources != nullptr);
+
+    const uint32_t counterCount = coverage->getCounterCount();
+    SLANG_CHECK_ABORT(counterCount > 0);
```

Cppcheck (2.21.0)

[style] The function 'makeArrayView' is never used.

(unusedFunction)